src/controllers/product.controller.js

```diff
@@ -273,15 +273,12 @@ const updateProduct = async (req, res) => {
     }
     const { name, price, quantity, bonus, expiryDate, categoryId } = req.body;
     await Products.update(
-      { name, price, quantity, categoryId, bonus, expiryDate },
+      { name, price, quantity, categoryId, bonus, expiryDate,images },
       { where: { id, sellerId: seller.id } }
```

Comment:
was it necessary to force seller to update  all fields of the Products like name, price, quantity, bonus etc all at the once whenever he/she want to make an update??? I think seller should update the some fields of his/her choice without changing all of them!!!

Reply:
thanks @katros1 , fixed